Distribution/Data/Magic/spells.json

```diff
@@ -2463,7 +2463,7 @@
     "Type": "Scripts.Zulu.Spells.Necromancy.DarknessSpell",
     "Name": "Darkness",
     "Mantra": "In Caligne Abditus",
-    "Circle": "Necro",
+    "Circle": "Lower lesser Chants of Necromancy",
```

Comment:
Just wondering why not use the ID rather here than the name which could be more prone to typos?

Reply:
Yeah true, I though about that and in the custom JSON converter you can actrually use either to reference the circle via Id or the string name. I just chose to use the string names here to be more self-explanatory than "21" etc.

Reply:
ok